test/remote/check_tests.js

```diff
@@ -28,9 +28,15 @@ test(
 
 var client = restify.createJsonClient({
   url: 'http://127.0.0.1:' + config.listen.port
-});
+})
+
+var ACTIONS = [
+  'accountCreate',
+  'accountLogin',
+  'passwordChange'
+]
 
-['accountCreate', 'accountLogin', 'passwordChange'].forEach(function (action) {
+ACTIONS.forEach(function (action) {
```

Comment:
I hated doing this, and I make myself sick, but ESLint was giving me the following error:
```
test/remote/check_tests.js
  33:1  error  Unexpected use of comma operator  no-sequences
```

I'll poke around at this some more and file an ESLint bug if needed since this definitely looks like a kosher usage of `.forEach()`. But for now I extracted into a separate array.